rtgui/editwidgets.h

```diff
@@ -570,6 +593,12 @@ inline Line::Line (int beginX, int beginY, int endX, int endY) :
         begin (beginX, beginY), end (endX, endY) {
 }
 
+inline Ellipse::Ellipse (rtengine::Coord& center, int radYT, int radY, int radXL, int radX,
```

Comment:
@Pandagrapher Salut Pierre, I really like the way you code: It's a delight to read. :+1:

Here's one place (not your fault) that needs to be changed: `center` isn't an "out" parameter and should be `const rtengine::Coord&`.

Reply:
@Floessie 
Sorry for my late answer... Thanks for the review :) I have corrected it with commit `5a58d32`
Pierre